File: src/main/webapp/resources/js/model/Word.js
function Word(wordId, word, partOfSpeech, frequency){
	this.wordId = wordId;
	this.word = word;
	this.partOfSpeech = partOfSpeech;
	this.frequency = frequency;
}

Word.prototype.getWordId = function(){
    return this.wordId;
}

Word.prototype.setWordId = function(wordId){
    this.wordId = wordId;
}

Word.prototype.getWord = function(){
    return this.word;
}

Word.prototype.setWord = function(word){
    this.word = word;
}

Word.prototype.getPartOfSpeech = function(){
    return this.partOfSpeech;
}

Word.prototype.setPartOfSpeech = function(partOfSpeech){
    this.partOfSpeech = partOfSpeech;
}

Word.prototype.getFrequency = function(){
    return this.frequency;
}

Word.prototype.setFrequency = function(frequency){
    this.frequency = frequency;
}

File: src/main/java/com/lingvoterra/dao/words/Word.java
package com.lingvoterra.dao.words;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

import com.fasterxml.jackson.annotation.JsonProperty;

@Entity
public class Word {
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private int wordId;

	@JsonProperty
	private String word;

	@JsonProperty
	private String partOfSpeech;

	public Word() {
	}

	public Word(int wordId, String word, String partOfSpeech, int frequency) {
		super();
		this.wordId = wordId;
		this.word = word;
		this.partOfSpeech = partOfSpeech;
	}

	public int getWordId() {
		return wordId;
	}

	public void setWordId(int wordId) {
		this.wordId = wordId;
	}

	public String getWord() {
		return word;
	}

	public void setWord(String word) {
		this.word = word;
	}

	public String getPartOfSpeech() {
		return partOfSpeech;
	}

	public void setPartOfSpeech(String partOfSpeech) {
		this.partOfSpeech = partOfSpeech;
	}
}
